src/sentry/options/defaults.py

```diff
@@ -443,6 +443,12 @@
 # Drop delete_old_primary_hash messages for a particular project.
 register("reprocessing2.drop-delete-old-primary-hash", default=[])
 
+
+# Send transaction events to random Kafka partitions. Currently
+# this defaults to false as transaction events are partitioned the same
+# as errors (by project ID). Eventually we will flip the default.
+register("kafka.partition-transactions-randomly", default=False)
```

Comment:
I could be wrong but I vaguely remember that you have to separate the deploys between creating the option and using it in the logic, and maybe have ops do something in between.

Reply:
Since we don't really need to change this on the fly and it might cause issues, I might actually just change the code to only fetch the option once